frontend/views/components/ProfileCard.vue

```diff
@@ -0,0 +1,254 @@
+<template lang='pug'>
+tooltip(
+  :direction='direction'
+  :manual='true'
+  ref='tooltip'
+  :opacity='1'
+  :aria-label='L("Show profile")'
+)
+  slot
+
+  template(slot='tooltip')
+    .card.c-profile(v-if='profile' role='dialog' :aria-label='L("{username} profile", { username })')
+      .c-identity(:class='{notGroupMember: !isActiveGroupMember}')
+        avatar-user(:username='username' size='lg')
+        user-name(:username='username')
+
+      i18n.has-text-1(
+        tag='p'
+        v-if='!isActiveGroupMember'
+      ) No longer a member of the group
+
+      p.c-bio(v-if='profile.bio')
+        | {{profile.bio}}
+        i18n.c-bio-link.is-unstyled.link(
+          v-if='isSelf'
+          tag='button'
+          @click='openModal("UserSettingsModal")'
+          data-test='linkEditBio'
+        ) Edit bio
+
+      i18n.button.is-small.is-outlined.c-bio-button(
+        v-else-if='isSelf'
+        tag='button'
+        @click='openModal("UserSettingsModal")'
+        data-test='buttonEditBio'
+      ) Add a bio
+
+      div(v-if='hasIncomeDetails')
```

Comment:
We should show a user's payment details regardless of whether or not we've filled in our own details.